src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/InvalidConfigurationException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.config;
+
+public class InvalidConfigurationException extends RuntimeException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4498197)